src/main/java/com/yurjinia/project_structure/board/controller/BoardController.java

```diff
@@ -0,0 +1,56 @@
+package com.yurjinia.project_structure.board.controller;
+
+import com.yurjinia.project_structure.board.dto.BoardDTO;
+import com.yurjinia.project_structure.board.dto.UpdateBoardRequest;
+import com.yurjinia.project_structure.board.service.BoardService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/projects/{projectCode}/boards")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @PostMapping
+    public ResponseEntity<Void> createBoard(@Valid @RequestBody BoardDTO boardDTO, @PathVariable String projectCode) {
+        boardService.createBoard(boardDTO, projectCode);
+        return ResponseEntity.status(HttpStatus.CREATED).build();
+    }
+
+    //todo можливо він маю бути в ProjectController
+    @GetMapping
+    public ResponseEntity<List<BoardDTO>> getProjectBoards(@PathVariable String projectCode) {
+        List<BoardDTO> projectBoards = boardService.getProjectBoards(projectCode);
+        return ResponseEntity.ok(projectBoards);
+    }
+
+    @PutMapping("/{boardCode}")
+    public ResponseEntity<BoardDTO> updateBoard(
+            @Valid @RequestBody UpdateBoardRequest updateBoardRequest,
```

Comment:
Path Variables must be at the beginning, and Request Body at the end.

Reply:
Also put the proper formatting - after parenthesis you need to have a first variable.